src/components/settings/PushNotifications/hooks/useNotificationTracking.ts

```diff
@@ -0,0 +1,82 @@
+import { keys as keysFromIndexedDb, update as updateIndexedDb } from 'idb-keyval'
+import { useEffect, useMemo } from 'react'
+
+import {
+  _DEFAULT_WEBHOOK_TRACKING,
```

Comment:
It shouldn't be underscored if it's imported by other modules.

Reply:
Adjusted in 14353657ff6ef1b8676bf6e59784935bc6b3a955.